app/src/main/java/ch/epfl/skysync/components/ConfirmFlightUI.kt

```diff
@@ -53,13 +54,27 @@ fun confirmation(plannedFlight: PlannedFlight, confirmClick: () -> Unit) {
   val date: LocalDate = plannedFlight.date
   val timeSlot = plannedFlight.timeSlot
   val vehicles: List<Vehicle> = plannedFlight.vehicles
+  val meetupLocationPassenger = "Nancy"
```

Comment:
Weird to still have hardcoded values in this component but maybe it was already here

Reply:
Yes it was already there. I agree we will have to change it asap